node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -57,10 +67,133 @@ pub struct SetGasPriceMsg {
     pub gas_price: String,
 }
 
+pub trait BatchPayableTools<T>
+where
+    T: BatchTransport,
+{
+    fn sign_transaction(
+        &self,
+        transaction_params: TransactionParameters,
+        web3: &Web3<Batch<T>>,
+        key: &secp256k1secrets::key::SecretKey,
+    ) -> Result<SignedTransaction, Web3Error>;
+    fn enter_raw_transaction_to_batch(&self, signed_transactions: Bytes, web3: &Web3<Batch<T>>);
+    fn batch_wide_timestamp(&self) -> SystemTime;
+    fn send_new_payable_fingerprints_credentials(
+        &self,
+        batch_wide_timestamp: SystemTime,
+        new_pp_fingerprints_sub: &Recipient<NewPendingPayableFingerprints>,
+        payable_attributes: &[(H256, u128)],
```

Comment:
Could you put the words "hash" and "balance" somewhere in this parameter name to make it clearer?